packages/core-ethereum/crates/core-ethereum-indexer/src/handlers.rs

```diff
@@ -1440,11 +1444,11 @@ pub mod wasm {
         }
 
         fn own_channel_updated(&self, channel_entry: &ChannelEntry) {
-            self.js_own_channel_updated(channel_entry.clone())
+            self.js_own_channel_updated(*channel_entry)
```

Comment:
I recommend to clone stuff before passing it to closures

Reply:
IMHO this works exactly the same way, since `ChannelEntry` implements both `Copy` and `Clone` implicitly.